src/main/resources/templates/c_plus_plus/test/testUtils/AbstractTest.py

```
@@ -0,0 +1,269 @@
+                    self._timeout()
+                except Exception as e:
+                    self.__markAsFailed(f"'{self.name}' had an internal error. {e}.\nPlease report this to an instructor!")
+                    print_exc()
+                    self._onFailed()
+        else:
+            # Run without timeout:
+            try:
+                self._run()
+            except TestFailedError:
+                printTester(f"'{self.name}' failed.")
+            except Exception as e:
```

Ruff

Do not catch blind exception: `Exception`

(BLE001)